CompilerSource/backend/GameData.cpp

```diff
@@ -284,12 +278,7 @@ TimelineData::TimelineData(const ::Timeline &timeline):
 }
 
 ObjectData::ObjectData(const buffers::resources::Object &q, const std::string& name):
-  buffers::resources::Object(q), name(name) {
-  for (int i = 0; i < events_size(); ++i) {
-    auto* evt = mutable_events(i);
-    evt->set_code(Actions2Code(evt->actions()));
```

Comment:
What is calling Actions2Code, now?

Reply:
It was GameData, now it's `libEGM/gmk.cpp` and `libEGM/gmx.cpp` as soon as a project is loaded.

Reply:
I like it.